php-frontend/src/main/java/org/sonar/php/compat/CompatibilityHelper.java

```diff
@@ -21,29 +21,18 @@
 
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.batch.sensor.SensorContext;
-import org.sonar.api.utils.Version;
 import org.sonar.plugins.php.api.visitors.PhpFile;
 
 /**
  * {@link PhpFile} factory class using SQ runtime version to choose which class to instantiate.
  */
 public class CompatibilityHelper {
 
-  private static final Version V6_0 = Version.create(6, 0);
-  private static final Version V6_2 = Version.create(6, 2);
-
   private CompatibilityHelper() {
     // utility class, forbidden constructor
   }
 
   public static PhpFile phpFile(InputFile inputFile, SensorContext context) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "context". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)